<h2 class="ltx_title_subsection">What is the relative importance of
willow genotype vs. the biotic and abiotic environment in structuring
aboveground communities?</h2><div>To address this question, we visually surveyed plants for arthropods to
determine the abundances of different (morpho)species. For the ant-aphid
experiment, we surveyed arthropods on 5 different occasions between
early June and late July 2012. For the wind experiment, we surveyed
arthropods once at the end of July 2012 and then once a month in May,
June, and July of 2013. So that individuals were not counted twice
between sampling dates, we took the maximum abundance for each arthropod
(morpho)species from each plant across all sampling dates within each
year. This approach provides a conservative estimate of the total number
of individuals of each (morpho)species that occurred on individual
plants through the summer. Given the relatively low abundances of
individual (morpho)species, we grouped arthropods at the Family-level
for insects and at the Order-level for all other arthropods prior to
analyzing community composition (details in Statistical Analyses
section below).</div><div></div><h2 class="ltx_title_subsection">Do willow genetic and environmental
variation have different effects on above and belowground
communities?</h2><div>To address this question, we dug up the willows from the wind experiment
to sample fungal and bacterial communities associated with willow roots
in late July of 2013. We did not sample belowground communities of
plants in the ant-aphid experiment due to the high mortality of plants
in 2013. To sample these belowground communities, we removed willows
with the surrounding soil intact to preserve root systems, separated
shoots and roots, then brushed soil off root systems and stored roots in
separate plastic bags. Within 6 hours of excavation, root systems were
stored at 4°C. To process roots, we gently rinsed them in tap water
until free of visible soil. In order to randomly select roots for
molecular analysis, second order roots were cut up into 2 cm lengths,
spread out on a grid, and then – using a random number generator – a
total of 30 cm of root length was picked from numbered grid cells. These
random root subsamples were flash frozen in liquid N, and kept at -80°C
until DNA extraction. To increase efficiency of DNA extraction, roots
were physically disrupted with 2 beads per 2 mL tube (3.0 mm Yttria
stabilized Zirconia Grinding Media) for 30 seconds at 1500 strokes per
minute (SPEX SamplePrep 200 geno/grinder). Total DNA was extracted from
the ground, frozen root samples using MoBio PowerSoil 96 sample DNA
extraction kits following the manufacture’s instructions.</div><div></div><div class="ltx_p">To identify fungal and bacterial OTUs, we used custom
Illumina-compatible barcode primer sets ITS1f/ITS4 and 515f/806r <cite class="ltx_cite raw v1">\cite{Caporaso_2012}</cite> to amplify via PCR the fungal ITS1, 5.8S,
and ITS2 region of ribosomal DNA and the V4 region of bacterial 16S
ribosomal DNA from total root DNA extractions. Product quality was
assessed by gel electrophoresis. PCR products were cleaned with
house-made magnetic bead solution, quantified with a Qubit fluorometric
kit, then sample libraries were pooled at a bacteria:fungi concentration
ratio of 2:1. Pooled amplicon libraries were sequenced as single-index
(the reverse barcode was uniquely indexed) 300 base pair reads at
Stanford Functional Genomics Facility on one lane of an Illumina MiSeq.
Quality control of reads consisted of these steps: trimming bases with
quality score less than 20 phred; trimming sequenced adaptors; and
removing reads with average error rates greater than 0.25 using UPARSE <cite class="ltx_cite raw v1">\cite{Edgar_2013}</cite>. Only high quality, paired forward and reverse reads were
used for OTU clustering at 97% identity, then OTUs were checked for
chimeras against the GOLD 16s rRNA database <cite class="ltx_cite raw v1">\cite{reddy2014genomes}</cite> and UNITE fungal ITS database ver6_97_13.05.2014 <cite class="ltx_cite raw v1">\cite{koljalg2005unite}</cite> with
UPARSE. Taxonomy was assigned using the RDP Classifier <cite class="ltx_cite raw v1">\cite{wang2007naive}</cite> and UNITE (ver6_97_13.05.2014) in QIIME <cite class="ltx_cite raw v1">\cite{Caporaso_2010}</cite>. We normalized datasets to account for differences in each
sample’s library size (number of reads obtained for each sample).
Finally, we discarded some OTUs and samples based on the following
conditions: OTUs with no known taxonomy (any OTU that did not blast to
at least Kingdom Fungi, Bacteria or Archaea); root samples with fewer
than 6000 fungal reads and 9000 bacterial reads; and mitochondrial and
chloroplast OTUs.</div><div class="ltx_p"></div><h2 class="ltx_title_subsection">What is the relative importance of
heritable trait variation, phenotypic plasticity, and direct
environmental effects in structuring
communities?</h2><div>Teasing apart the relative importance of these factors requires
simultaneous measurements of plant functional traits, correlated
environmental gradients, and community-level patterns within a
genotype-by-environment experiment.</div><div></div><h3 class="ltx_title_subsubsection">Plant Traits</h3><div class="ltx_p">Prior work in this study system has demonstrated that variation in both plant growth and
leaf quality traits affect the likelihood of willows being colonized by
foliar arthropods <cite class="ltx_cite raw v1">\cite{Barbour_2015}</cite>. To quantify plant-growth
traits, we measured plant height, the number of shoots produced, and
average shoot length in late July of each year (end of growing season)
for both experiments. We quantified plant height as the distance (mm)
from the ground to the tip of the tallest shoot. We quantified average
shoot length by measuring every shoot on each plant to the nearest
millimeter and calculating the average shoot length for each plant. We
also measured several traits that could shape leaf quality for
herbivores, including water content, trichome density, specific leaf
area (SLA), percentage carbon (C) and nitrogen (N), and C:N. To measure
these traits, we excised fully expanded and undamaged leaves from plants
in late July of each year, stored leaf samples with a moist paper towel
in separate plastic bags within a cooler and immediately brought them
back to the laboratory. We then weighed leaves to obtain fresh mass (g),
digitally scanned them to measure leaf area (<span class="math ltx_Math v1">\(mm^2\)</span>)
using ImageJ <cite class="ltx_cite raw v1">\cite{abramoff2004image}</cite>, and oven-dried them at
60 °C for 72 h to obtain dry weight (g)<cite class="ltx_cite raw v1">\cite{cornelissen2003handbook}</cite>. We calculated SLA as</div><div><span class="math ltx_Math v1">\(\frac{leaf\ area}{dry\ mass}\)</span> <cite class="ltx_cite raw v1">\cite{cornelissen2003handbook}</cite>. We calculated leaf water content as
the&nbsp;<span class="math ltx_Math v1">\begin{equation}
\frac{(fresh\ mass-dry\ mass)}{\text{dry\ mass}}\nonumber \\
 \end{equation}</span> (Munns &amp; PrometheusWiki Contributors 2010). To measure trichome
density, we counted the number of trichomes along an 11 mm by 1 mm
transect in the center of the leaf, halfway between the leaf edge and
the mid-vein, under a dissecting scope. To measure percentage C and N,
we ground oven-dried leaves to a fine powder using a ball mill
(Mixer/Mill 8000D, SPEX SamplePrep; Metuchen, NJ, USA). Subsamples of
each material were then analyzed for percentage C and N on an elemental
analyzer (ECS 4010; Costech Analytical Technologies, Valencia,
California, USA) using atropine (4.84% N and 70.56% C) as a reference
standard. For root-associated communities, we hypothesized that
variation in root C:N may affect community assembly. To measure root C
and N, a subsample of oven-dried roots were crushed with a razor blade,
then analyzed for percent C and N on an elemental analyzer (Carlo-Erba
NA 1500) using atropine (4.84% N and 70.56% C) as a reference
standard.</div><div></div><h3 id="auto-label-subsubsection-721519" class="ltx_title_subsubsection">Soil
characteristics</h3><div>Soil nutrients, total organic matter, and moisture
may all influence plant traits and the assembly of fungal and bacterial
communities on plant roots <cite class="ltx_cite raw v1">\cite{Erlandson_2015}</cite>. Moreover, we
expected that wind exposure would affect these soil characteristics
(Lortie &amp; Cushman 2007); therefore, we measured soil nutrients, percent
organic matter, and moisture within each plot of the wind experiment
(one exposed and one unexposed plot per block).
\label{soil-characteristics-soil-nutrients-total-organic-matter-and-moisture-may-all-influence-plant-traits-and-the-assembly-of-fungal-and-bacterial-communities-on-plant-roots-erlandson-et-al.-2015.-moreover-we-expected-that-wind-exposure-would-affect-these-soil-characteristics-lortie-cushman-2007-therefore-we-measured-soil-nutrients-percent-organic-matter-and-moisture-within-each-plot-of-the-wind-experiment-one-exposed-and-one-unexposed-plot-per-block.}</div><div class="ltx_p">To estimate soil nutrient uptake by willows, we installed Plant Root
Simulator (PRS) Probes (Western Ag Innovations, Saskatchewan, Canada) at
three randomly selected locations within each plot for 11 days in
September 2012. PRS Probes estimate nutrient supply rates to roots by
continuously adsorbing charged ionic elements over the burial period.
For our study, we estimated potential root uptake of
NO3+,
NH4-, Ca, Mg, K, P, Fe, Mn, Cu, Zn, B,
S, Pb, Al, and Cd. From this nutrient data, we calculated total N as
NO3+ +
NH4-, and then used principal
components analysis to condense these nutrients into a single axis
(nutrients PC1) that explained 34% of the variation. Nutrients PC1
described the negative correlation between nitrogen compounds
(NO3+,
NH4-) and the rest of the ionic
elements, with positive values indicating high supply rates of all ionic
elements except for the nitrogen compounds. To measure percent organic
matter content (%OM), we used a trowel to collect soil (depth = 0 – 15
cm) adjacent to the randomly positioned PRS probes in September 2012.
Soils were transported back to the lab in plastic bags, sieved into
fragments less than 2 mm, randomly subsampled using a soil splitter, and
dried at 105 <b>°</b>C for 72 hours. We then weighed a subsample of
the oven dried soil into an oven dried crucible and placed the crucible
and soil into a furnace to be combusted at 375 <b>°</b>C for 16 hours.
We then weighed the combusted samples, placed them in a desiccator for
20 minutes, and weighed them again. We calculated percent organic matter
as</div><div><span class="math ltx_Math v1">\begin{equation}
\%OM=\ \frac{\text{Oven\ dry\ mass\ }(g)\ \ Combusted\ Mass\ (g)}{Oven\ Dry\ Mass\ (g)}\times 100\nonumber \\
 \end{equation}</span></div><div class="ltx_p">. To measure soil moisture (volumetric water content,
m3/m3), we used a 5TE soil sensor
coupled to an EM50 Digital/Analog Data Logger (Decagon Devices, Pullman,
Washington, USA). In September 2012, while PRS probes were in the
ground, we measured soil moisture at a depth of 5 cm in three random
locations within each plot on three different days between 1100 – 1500
hours. We repeated this same sampling scheme in early July 2013. Plot
levels measurements of soil moisture were highly correlated between
years (Pearson’s r = 0.93, t18 = 10.91, P
&lt; 0.001), so we averaged these soil moisture estimates to
determine a single soil moisture value per plot.</div>