<div class="ltx_p"><b>Abstract tes</b></div><div class="ltx_p"></div><div class="ltx_p">1.</div><ol><li>Host-plant genetic variation affects the diversity and composition of
associated above and belowground communities. Most evidence supporting
this view is derived from studies within a single common garden,
thereby constraining the range of biotic and abiotic environmental
conditions that might directly or indirectly (via phenotypic
plasticity) affect communities. If natural variability in the
environment renders host-plant genetic effects on associated
communities unimportant, then studying the community-level
consequences of genetic variation may not be warranted.</li><li>We addressed this knowledge gap by planting a series of common gardens
consisting of 10 different clones (genotypes) of the willow
Salix hookeriana in a coastal dune ecosystem and manipulated
natural variation in ant-aphid interactions (biotic) and wind exposure
(abiotic) in two separate experiments. We then quantified the
responses of associated species assemblages both above (foliar
arthropods) and belowground (rhizosphere fungi and bacteria). In
addition, we quantified plant phenotypic responses (plant growth, leaf
quality, and root quality) to tease apart the effects of heritable
trait variation, phenotypic plasticity, and direct environmental
effects on associated communities.</li><li>In the ant-aphid experiment, we found that willow genotype explained
more variation in foliar arthropod communities than aphid additions
and proximity to aphid-tending ant mounds. However, aphid additions
modified willow genotype effects on arthropod community composition by
attracting other aphid species to certain willow genotypes. In the
wind experiment, wind exposure explained more variation than willow
genotype in communities of foliar arthropods and root bacteria. Still,
willow genotype had strong effect sizes on several community
properties of arthropods and fungi, indicating that host-plant genetic
variation remains important. In both experiments, plant-growth traits
were the primary determinants of arthropod community structure with
leaf quality traits playing a lesser role; however, we failed to
identify which root traits were mediating belowground community
responses.</li><li>Taken together, our results suggest that host-plant genetic variation is
an important driver of above and belowground biodiversity, despite
natural variation in the biotic and abiotic environment.</li><li></li></ol><h3 class="ltx_title_subsubsection"></h3><div></div><h1 id="auto-label-section-761486" class="ltx_title_section">Introduction</h1><div>Intraspecific genetic variation is a key driver of trait variation
within host plants, which in turn can have cascading effects on
associated species and entire communities of organisms <cite class="ltx_cite raw v1">\cite{Fritz_1988}</cite><cite class="ltx_cite raw v1">\cite{Lamit_2016}</cite> a<cite class="ltx_cite raw v1">\cite{Maddox_1990}</cite>(Fritz &amp; Price
1988; Maddox &amp; Root 1990; Antonovics 1992; Lamit et al. 2015).
For example, genetic variation in the leaf chemistry of cottonwoods
(Whitham et al. 2006) and in the plant architecture of coyote
bush (Crutsinger et al. 2014) structures diverse assemblages of
species, from foliar arthropods aboveground to soil microbes below.
While community-level consequences of genetic variation (commonly
referred to as ‘community genetics’, sensu Antonovics 1992) have
been documented in a variety of host-plant taxa (Whitham et al.
2012), evidence comes primarily from common garden experiments that
minimize environmental variation. These controlled environments likely
limit effects of the biotic and abiotic environment on the expression of
host plant traits (Gratani 2014) as well as the diversity and
composition of species assemblages (MacArthur 1972; Gaston 2000).
Therefore, the importance of host-plant genetic variation in structuring
ecological communities in naturally varying environments remains unclear
(Hersch-Green, Turley &amp; Johnson 2011; Tack, Johnson &amp; Roslin 2012;
Crutsinger 2016).</div><div class="ltx_p">A key challenge for advancing studies of community genetics beyond the
common garden is to identify important biotic and abiotic environmental
factors that structure communities associated with host plants. For
example, a series of experiments in common milkweed (Asclepias
syriaca) have shown that a diversity of biotic factors, such as light
competition from neighboring plants (Agrawal &amp; Zandt 2003), caterpillar
herbivory (Abdala‐Roberts, Agrawal &amp; Mooney 2012), and aphid-tending
ants (Mooney &amp; Agrawal 2008; Abdala-Roberts et al. 2012) can act
independently or interact with milkweed genotype to shape its associated
community of foliar arthropods. Similarly, abiotic factors, such as soil
nutrient availability, can act independently or modify the effects of
host-plant genotype on herbivore assemblages (Orians &amp; Fritz 1998) and
tri-trophic interactions (Rossi &amp; Stiling 1998; Abdala-Roberts &amp;
Mooney 2013). Still, we are lacking explicit comparisons of host-plant
genotypic effects across natural gradients in both biotic and abiotic
factors, so their relative importance in structuring communities is
unclear.</div><div class="ltx_p">Although host plants provide essential resources for a diverse array of
taxa both above- and belowground, the majority of community genetics
studies have focused on aboveground assemblages (Whitham et al.
2012). Studies that have simultaneously examined above- and belowground
communities have found variable results, with host-plant genetic effects
on aboveground communities being stronger (Crutsinger et al.
2008; Bailey et al. 2009) or comparable and coupled (Crutsinger
et al. 2014) with those belowground. Above- and belowground
linkages can have important consequences for both plant fitness (Whitham
et al. 2006) and terrestrial ecosystem processes (Wardle et
al. 2004). In addition, feedbacks between above- and belowground
assemblages may depend strongly on the biotic and/or abiotic environment
(Wardle et al. 2004). Consequently, a rising challenge for
community genetics is to understand the linkages between above- and
belowground communities (Crutsinger et al. 2014; Lamit et
al. 2015) and whether these linkages are modified by environmental
variation.</div><div class="ltx_p">Host-plant traits determine the quantity and quality of resources for
the diverse organisms that colonize them; therefore, measuring
functional trait responses of host-plant genotypes to different
environments can give insight to mechanisms of community assembly in
genotype-by-environment studies. Phenotypic traits can vary in their
plasticity (change in trait expression of a genotype in response to the
environment: Scheiner 1993) and may even be plastic in response to one
environmental gradient but not another (Scheiner &amp; Goodnight 1984,
Garbutt &amp; Bazzaz 1987, Scheiner 1993). In addition, multiple plant
traits can be important in structuring associated communities on host
plants (Agrawal 2004, 2005; Agrawal &amp; Fishbein 2006; Barbour et al.
2015, 2016). Simultaneous measurements of multiple functional traits and
community-level patterns in genotype-by-environment studies can
distinguish the effects of heritable trait variation (proportion of
variance in a trait explained by genotype: Lynch &amp; Walsh 1998),
phenotypic plasticity, and direct environmental effects on species
assemblages.</div><div class="ltx_p">Here, we use common garden experiments to examine how host-plant
genotypic variation as well as the biotic and abiotic environment
structure communities associated with the willow Salix hookeriana
in a coastal dune ecosystem. Prior work in this system has shown that
willow genotypes host distinct arthropod communities and that multiple
plant traits are important in determining community assembly (Barbour
et al. 2015; 2016). Importantly, these traits varied
substantially in their degree of heritability (plant growth, mean
H 2 = 0.26; leaf quality, mean
H 2 = 0.72), suggesting that the environment may
influence them in different ways. We sought to answer the following
questions: (1) What is the relative importance of willow genotype vs.
the biotic and abiotic environment in structuring aboveground
communities? (2) Do willow genetic and environmental variation have
different effects on above and belowground communities? (3) What is the
relative importance of heritable trait variation, phenotypic plasticity,
and direct environmental effects in structuring communities?</div><h3 class="ltx_title_subsubsection">Materials and Methods</h3><div>\label{materials-and-methods}</div><div class="ltx_title_paragraph">Study
site</div><div>\label{study-site}</div><div class="ltx_p">We conducted this research at Lanphere Dunes (40<b>°</b> 53’29.85”N,
124<b>°</b> 8’49.06”W), a pristine coastal dune ecosystem managed by
US Fish and Wildlife service in Humboldt County, California, USA.
Coastal willow (Salix hookeriana ex Barratt ex Hooker) naturally
occurs in nearshore dune swales – seasonal freshwater wetlands that
form in depressions between dune ridges (Pickart 2009). Aside from
coastal willow (hereafter willow), the dominant vegetation in these
swales consists of beach pine (Pinus contorta ssp.
Contorta ) and slough sedge (Carex obnupta).</div><div class="ltx_p">During preliminary surveys, we qualitatively identified two important
sources of environmental variation for willows in the dunes – one
biotic (the presence of ant-aphid mutualisms) and one abiotic (wind
exposure). We observed that the aphid Aphis farinosa was an
abundant herbivore at Lanphere Dunes. Aphis farinosa is usually
found at the tips of new shoot growth where it feeds on willow phloem.
As with many other aphid species, A. farinosa excretes
carbohydrate-rich honeydew while feeding, which attracts ants that tend
the aphids and feed on the honeydew. This ant-aphid interaction is often
mutualistic, because the ants will defend aphids from predatory
arthropods and also eat other herbivores that may be competing with the
aphids (Floate &amp; Whitham 1994; Mooney &amp; Agrawal 2008). The ant species
we observed most frequently tending A. farinosa was the western
thatching ant, Formica obscuripes. Western thatching ants create
distinct dome-shaped mounds from nearby plant-material and are known to
reduce herbivory from leaf chewing arthropods on S. hookeriana at
our study site (Crutsinger &amp; Sanders 2005), presumably by deterring
ovipositing females or predating young larva. This work suggests that
the presence of aphids and the proximity to ant mounds influences
associated communities through three non-mutually exclusive mechanisms:
(i) increased abundance of aphid-tending ants, which could deter other
arthropods; (ii) attraction of predators or deterrence of other
herbivores, by aphids; (iii) alteration of plant-growth or leaf quality
traits by aphids. We also observed that willows growing in wind-exposed
habitats often exhibit reduced growth, especially at their leading edge,
appearing to be “swept back” by the wind. We hypothesized that wind
exposure may influence associated communities through three non-mutually
exclusive mechanisms: (i) reduced plant size due to wind pruning; (ii)
altered soil characteristics due to increased evaporation; and (iii)
direct inhibition of ovipositing female arthropods.</div><div class="ltx_title_paragraph">Experimental
design</div><div>\label{experimental-design}</div><div class="ltx_p">Prior to bud break in February 2012, we took shoot cuttings (40 cm
length &amp; ~0.5 cm diameter) from one to two replicates
of 10 different willow genotypes from a pool of 26 locally collected
willow genotypes planted in a large common garden experiment. Details
about the establishment of this common garden are given in Barbour
et al. (2015). These 10 genotypes displayed substantial variation
in both plant-growth and leaf-quality traits (Barbour et al.
2015). Shoot cuttings were soaked in water overnight and then planted in
a mixture of 80% perlite, 20% peat moss (dolomite lime added to
balance pH) inside “Cone-tainers”™ (Stuewe &amp; Sons, Inc.). We grew
cuttings under ambient weather conditions outside the greenhouse at
Humboldt State University until we transplanted willows into multiple
common gardens at Lanphere Dunes.</div><div>\label{section-1}



Ant-aphid
experiment\label{ant-aphid-experiment}</div><div class="ltx_p">To examine how the presence of aphids, proximity to ant mounds, and
willow genotype affected associated communities, we established common
gardens around 5 different ant mounds (treated as blocks) in late May
2012. Within each block, we randomly planted 20 cuttings (2 replicates
of each of 10 genotypes) with 0.5 m spacing in plots that were at a
distance of 1, 6, and 12 meters from the edge of the ant mound, for a
total of 60 cuttings per ant mound (300 cuttings for entire experiment).
Within each plot, we randomly assigned the aphid treatment (aphid
presence vs. absence) to one of the two replicates for each genotype. On
May 22, we collected aphids (Aphis farinosa) from a single willow
patch at Lanphere Dunes and placed 5 adult apterate aphids on the tips
of willow cuttings in the aphid treatment using a moist paintbrush. We
bagged aphids onto the apical shoots of cuttings using organza bags to
promote aphid establishment on plants. Similarly, we placed organza bags
on all control plants. On May 27, we checked aphid treatments to ensure
there were 5 adult aphids and removed bags from all cuttings. If
necessary, we added aphids to these treatments until there were 5 adults
and we removed any aphid nymphs that were produced since initial
establishment. We checked plants for aphids on June 6, June 13, June 24,
July 4, July 14, and July 20, 2012. If plants in the aphid treatment had
less than 5 apterate aphids, we noted their abundance and added aphids
until there were at least 5 individuals. The ant-aphid experiment was
restricted to the summer of 2012, because in the summer of 2013 there
was high willow mortality induced by drought and A. farinosa was
too low in abundance on naturally occurring willows to allow us to
repeat the experiment.</div><div>Wind
experiment\label{wind-experiment}</div><div class="ltx_p">To examine how wind exposure and willow genotype affected associated
communities in the coastal dunes, we planted 200 willow cuttings in a
split-plot experimental design in late May of 2012. At each of 10
different willow patches (treated as blocks), we established an
‘exposed’ and a ‘unexposed’ common garden with exposed gardens facing
prevailing winds during the growing season. Each garden consisted of one
replicate cutting of each of 10 genotypes randomly planted in 2 m by 0.5
m grid with 0.5 m spacing between plants. The center of exposed and
unexposed gardens within each block were the same distance (2 m) from
the edge of the willow patch to control for insect accessibility. To
estimate the difference in wind conditions experienced by exposed vs.
unexposed plants, we went out on a representative windy afternoon in
September 2012. A nearby weather station estimated wind speeds of 22
km/h during this period (Arcata, CA). We used a hand-held anemometer
(Kestrel 1000) to measure wind speed at a height of 37 cm aboveground
(approximate height of tallest plants in the garden in 2012) in each
plot of our experiment. For each block, we randomly selected the order
in which exposed and unexposed plots were measured and took maximum wind
speed measurements over a 30 s period. We found that willows growing in
wind-exposed plots experienced up to 3.7-fold higher wind speeds
compared to unexposed plots (F1,9 = 187.32, P
&lt; 0.001), suggesting that the location of our plots were
effective manipulations of wind exposure.</div><div class="ltx_title_paragraph">What is the relative importance of
willow genotype vs. the biotic and abiotic environment in structuring
aboveground communities?</div><div>\label{what-is-the-relative-importance-of-willow-genotype-vs.-the-biotic-and-abiotic-environment-in-structuring-aboveground-communities}</div><div class="ltx_p">To address this question, we visually surveyed plants for arthropods to
determine the abundances of different (morpho)species. For the ant-aphid
experiment, we surveyed arthropods on 5 different occasions between
early June and late July 2012. For the wind experiment, we surveyed
arthropods once at the end of July 2012 and then once a month in May,
June, and July of 2013. So that individuals were not counted twice
between sampling dates, we took the maximum abundance for each arthropod
(morpho)species from each plant across all sampling dates within each
year. This approach provides a conservative estimate of the total number
of individuals of each (morpho)species that occurred on individual
plants through the summer. Given the relatively low abundances of
individual (morpho)species, we grouped arthropods at the Family-level
for insects and at the Order-level for all other arthropods prior to
analyzing community composition (details in Statistical Analyses
section below).</div><div class="ltx_title_paragraph">Do willow genetic and environmental
variation have different effects on above and belowground
communities?</div><div>\label{do-willow-genetic-and-environmental-variation-have-different-effects-on-above-and-belowground-communities}</div><div class="ltx_p">To address this question, we dug up the willows from the wind experiment
to sample fungal and bacterial communities associated with willow roots
in late July of 2013. We did not sample belowground communities of
plants in the ant-aphid experiment due to the high mortality of plants
in 2013. To sample these belowground communities, we removed willows
with the surrounding soil intact to preserve root systems, separated
shoots and roots, then brushed soil off root systems and stored roots in
separate plastic bags. Within 6 hours of excavation, root systems were
stored at 4°C. To process roots, we gently rinsed them in tap water
until free of visible soil. In order to randomly select roots for
molecular analysis, second order roots were cut up into 2 cm lengths,
spread out on a grid, and then – using a random number generator – a
total of 30 cm of root length was picked from numbered grid cells. These
random root subsamples were flash frozen in liquid N, and kept at -80°C
until DNA extraction. To increase efficiency of DNA extraction, roots
were physically disrupted with 2 beads per 2 mL tube (3.0 mm Yttria
stabilized Zirconia Grinding Media) for 30 seconds at 1500 strokes per
minute (SPEX SamplePrep 200 geno/grinder). Total DNA was extracted from
the ground, frozen root samples using MoBio PowerSoil 96 sample DNA
extraction kits following the manufacture’s instructions.</div><div class="ltx_p">To identify fungal and bacterial OTUs, we used custom
Illumina-compatible barcode primer sets ITS1f/ITS4 and 515f/806r
(Caporaso et al. 2012) to amplify via PCR the fungal ITS1, 5.8S,
and ITS2 region of ribosomal DNA and the V4 region of bacterial 16S
ribosomal DNA from total root DNA extractions. Product quality was
assessed by gel electrophoresis. PCR products were cleaned with
house-made magnetic bead solution, quantified with a Qubit fluorometric
kit, then sample libraries were pooled at a bacteria:fungi concentration
ratio of 2:1. Pooled amplicon libraries were sequenced as single-index
(the reverse barcode was uniquely indexed) 300 base pair reads at
Stanford Functional Genomics Facility on one lane of an Illumina MiSeq.
Quality control of reads consisted of these steps: trimming bases with
quality score less than 20 phred; trimming sequenced adaptors; and
removing reads with average error rates greater than 0.25 using UPARSE
(Edgar 2013). Only high quality, paired forward and reverse reads were
used for OTU clustering at 97% identity, then OTUs were checked for
chimeras against the GOLD 16s rRNA database (Reddy et al. 2015)
and UNITE fungal ITS database ver6_97_13.05.2014 (Kõljalg et al.) with
UPARSE. Taxonomy was assigned using the RDP Classifier (Wang et
al. 2007) and UNITE (ver6_97_13.05.2014) in QIIME (Caporaso et
al. 2010). We normalized datasets to account for differences in each
sample’s library size (number of reads obtained for each sample).
Finally, we discarded some OTUs and samples based on the following
conditions: OTUs with no known taxonomy (any OTU that did not blast to
at least Kingdom Fungi, Bacteria or Archaea); root samples with fewer
than 6000 fungal reads and 9000 bacterial reads; and mitochondrial and
chloroplast OTUs.</div><div class="ltx_title_paragraph">What is the relative importance of
heritable trait variation, phenotypic plasticity, and direct
environmental effects in structuring
communities?</div><div>\label{what-is-the-relative-importance-of-heritable-trait-variation-phenotypic-plasticity-and-direct-environmental-effects-in-structuring-communities}</div><div class="ltx_p">Teasing apart the relative importance of these factors requires
simultaneous measurements of plant functional traits, correlated
environmental gradients, and community-level patterns within a
genotype-by-environment experiment.</div><div class="ltx_p">study system has demonstrated that variation in both plant growth and
leaf quality traits affect the likelihood of willows being colonized by
foliar arthropods (Barbour et al. 2015). To quantify plant-growth
traits, we measured plant height, the number of shoots produced, and
average shoot length in late July of each year (end of growing season)
for both experiments. We quantified plant height as the distance (mm)
from the ground to the tip of the tallest shoot. We quantified average
shoot length by measuring every shoot on each plant to the nearest
millimeter and calculating the average shoot length for each plant. We
also measured several traits that could shape leaf quality for
herbivores, including water content, trichome density, specific leaf
area (SLA), percentage carbon (C) and nitrogen (N), and C:N. To measure
these traits, we excised fully expanded and undamaged leaves from plants
in late July of each year, stored leaf samples with a moist paper towel
in separate plastic bags within a cooler and immediately brought them
back to the laboratory. We then weighed leaves to obtain fresh mass (g),
digitally scanned them to measure leaf area (mm2)
using ImageJ (Abrámoff, Magalhães, and Ram 2004), and oven-dried them at
60 °C for 72 h to obtain dry weight (g) (Cornelissen et al.
2003). We calculated SLA as</div><div><span class="math ltx_Math v1">\begin{equation}
\frac{\text{leaf\ area}}{\text{dry\ mass}}\nonumber \\
 \end{equation}</span></div><div class="ltx_p">(Cornelissen et al. 2003). We calculated leaf water content as
the</div><div><span class="math ltx_Math v1">\begin{equation}
\frac{(fresh\ mass-dry\ mass)}{\text{dry\ mass}}\nonumber \\
 \end{equation}</span></div><div class="ltx_p">(Munns &amp; PrometheusWiki Contributors 2010). To measure trichome
density, we counted the number of trichomes along an 11 mm by 1 mm
transect in the center of the leaf, halfway between the leaf edge and
the mid-vein, under a dissecting scope. To measure percentage C and N,
we ground oven-dried leaves to a fine powder using a ball mill
(Mixer/Mill 8000D, SPEX SamplePrep; Metuchen, NJ, USA). Subsamples of
each material were then analyzed for percentage C and N on an elemental
analyzer (ECS 4010; Costech Analytical Technologies, Valencia,
California, USA) using atropine (4.84% N and 70.56% C) as a reference
standard. For root-associated communities, we hypothesized that
variation in root C:N may affect community assembly. To measure root C
and N, a subsample of oven-dried roots were crushed with a razor blade,
then analyzed for percent C and N on an elemental analyzer (Carlo-Erba
NA 1500) using atropine (4.84% N and 70.56% C) as a reference
standard.</div><div>Soil
characteristics – Soil nutrients, total organic matter, and moisture
may all influence plant traits and the assembly of fungal and bacterial
communities on plant roots (Erlandson et al. 2015). Moreover, we
expected that wind exposure would affect these soil characteristics
(Lortie &amp; Cushman 2007); therefore, we measured soil nutrients, percent
organic matter, and moisture within each plot of the wind experiment
(one exposed and one unexposed plot per block).
\label{soil-characteristics-soil-nutrients-total-organic-matter-and-moisture-may-all-influence-plant-traits-and-the-assembly-of-fungal-and-bacterial-communities-on-plant-roots-erlandson-et-al.-2015.-moreover-we-expected-that-wind-exposure-would-affect-these-soil-characteristics-lortie-cushman-2007-therefore-we-measured-soil-nutrients-percent-organic-matter-and-moisture-within-each-plot-of-the-wind-experiment-one-exposed-and-one-unexposed-plot-per-block.}</div><div class="ltx_p">To estimate soil nutrient uptake by willows, we installed Plant Root
Simulator (PRS) Probes (Western Ag Innovations, Saskatchewan, Canada) at
three randomly selected locations within each plot for 11 days in
September 2012. PRS Probes estimate nutrient supply rates to roots by
continuously adsorbing charged ionic elements over the burial period.
For our study, we estimated potential root uptake of
NO3+,
NH4-, Ca, Mg, K, P, Fe, Mn, Cu, Zn, B,
S, Pb, Al, and Cd. From this nutrient data, we calculated total N as
NO3+ +
NH4-, and then used principal
components analysis to condense these nutrients into a single axis
(nutrients PC1) that explained 34% of the variation. Nutrients PC1
described the negative correlation between nitrogen compounds
(NO3+,
NH4-) and the rest of the ionic
elements, with positive values indicating high supply rates of all ionic
elements except for the nitrogen compounds. To measure percent organic
matter content (%OM), we used a trowel to collect soil (depth = 0 – 15
cm) adjacent to the randomly positioned PRS probes in September 2012.
Soils were transported back to the lab in plastic bags, sieved into
fragments less than 2 mm, randomly subsampled using a soil splitter, and
dried at 105 <b>°</b>C for 72 hours. We then weighed a subsample of
the oven dried soil into an oven dried crucible and placed the crucible
and soil into a furnace to be combusted at 375 <b>°</b>C for 16 hours.
We then weighed the combusted samples, placed them in a desiccator for
20 minutes, and weighed them again. We calculated percent organic matter
as</div><div><span class="math ltx_Math v1">\begin{equation}
\%OM=\ \frac{\text{Oven\ dry\ mass\ }(g)\ \ Combusted\ Mass\ (g)}{Oven\ Dry\ Mass\ (g)}\times 100\nonumber \\
 \end{equation}</span></div><div class="ltx_p">. To measure soil moisture (volumetric water content,
m3/m3), we used a 5TE soil sensor
coupled to an EM50 Digital/Analog Data Logger (Decagon Devices, Pullman,
Washington, USA). In September 2012, while PRS probes were in the
ground, we measured soil moisture at a depth of 5 cm in three random
locations within each plot on three different days between 1100 – 1500
hours. We repeated this same sampling scheme in early July 2013. Plot
levels measurements of soil moisture were highly correlated between
years (Pearson’s r = 0.93, t18 = 10.91, P
&lt; 0.001), so we averaged these soil moisture estimates to
determine a single soil moisture value per plot.</div>