---
title: "Lanphere Dunes OTU table processing"
author: "SRE"
date: "July 12, 2016"
output: word_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```



```{r load packages and biom files}
library('phyloseq')
library("ggplot2")
library('metagenomeSeq')
library("biomformat")
hf<-import_biom("/Users/sonyaerlandson/Documents/Chapter3/Data/otu_table_wTax_ITS.biom", parseFunction = parse_taxonomy_greengenes) 
hf

sampledata<-read.csv("/Users/sonyaerlandson/Documents/Chapter3/Data/hum_sampledata_all.csv", header=TRUE)
cn<-read.csv("/Users/sonyaerlandson/Documents/Chapter3/Data/RootCN.csv", header=TRUE)

samplenamesfun<-sample_names(hf)
sf<-as.character(sampledata$ID) %in% sample_names(hf)    
samf<-sampledata[sf, ]
row.names(samf)<-samf$ID
samf<-data.frame(samf)
sample_data(hf)<-samf
hf
hf0<-prune_taxa(taxa_sums(hf) != 0, hf)
```
hf
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 2009 taxa and 134 samples ]
sample_data() Sample Data:       [ 134 samples by 7 sample variables ]
tax_table()   Taxonomy Table:    [ 2009 taxa by 8 taxonomic ranks ]


```{r}
#remove taxa that are not matched to kingdom fungi
kingdom<-as.vector(data.frame(tax_table(hf0))$Kingdom)
unique(kingdom)
tax.keep<-kingdom =="Fungi"
tax.keep[is.na(tax.keep)]<-FALSE
hf1<-prune_taxa(tax.keep,hf0)

#prev0 = apply(X = otu_table(hf1),
#                MARGIN = ifelse(taxa_are_rows(hf1), yes = 1, no = 2),
#                FUN = function(x){sum(x > 0)})
#prevdf = data.frame(Prevalence = prev0,
#                      TotalAbundance = taxa_sums(hf1),
#                      tax_table(hf1))
#keepClass = table(prevdf$Class)[(table(prevdf$Class) > 4)]
#prevdf1 = subset(prevdf, Class %in% names(keepClass))
# Define prevalence threshold as 5% of total samples
#prevalenceThreshold = 0.05 * nsamples(hf1)
#prevalenceThreshold

# Execute prevalence filter, using `prune_taxa()` function
#hf2 = prune_taxa((prev0 >= prevalenceThreshold), hf1)
hf1
sort(sample_sums(hf1))

hf1<-prune_samples(sample_sums(hf1)>10, hf1)
hf2

tax_table(hf1)<-tax_table(hf1)[ ,-8]
hf1
unique(tax_table(hf1)[ ,5])
rank_names(hf1)

#phyloseq-class experiment-level object
#otu_table()   OTU Table:         [ 544 taxa and 133 samples ]
#sample_data() Sample Data:       [ 133 samples by 7 sample variables ]
#tax_table()   Taxonomy Table:    [ 544 taxa by 7 taxonomic ranks ]
 

hfg<-read.delim("/Users/sonyaerlandson/Documents/Chapter3/Data/humfungiOTUtableFUNGUILD.guilds.txt", header=TRUE, sep = "\t")
guilds<-hfg[ ,c(139:142)]
row.names(guilds)<-hfg$OTU_ID

#ecto
hfecm.guild<-read.csv("/Users/sonyaerlandson/Documents/Chapter3/Data/fungi_FUNGUILD_ECM.csv", header=TRUE)

#amf
hfamf.guild<-read.csv("/Users/sonyaerlandson/Documents/Chapter3/Data/fungi_FUNGUILD_AMF.csv", header = TRUE)

#pathogens
hfpath.guild<-read.csv("/Users/sonyaerlandson/Documents/Chapter3/Data/fungi_FUNGUILD_pathogens.csv", header = TRUE)

# put guild OTUs in same order as phyloseq OTUs
guilds<-guilds[match(row.names(tax_table(hf2)),row.names(guilds)), ]
identical(row.names(tax_table(hf2)), row.names(guilds))

#add guild information to taxonomy table
taxfinal<-data.frame(tax_table(hf2))
taxfinal<-cbind(taxfinal, guilds[ ,c(3,4)])
head(taxfinal)
taxfinaltable<-tax_table(as.matrix(taxfinal))
tax_table(hf2)<-tax_table(taxfinaltable)

hf3<-phyloseq_to_metagenomeSeq(hf2)

hf4<-MRexperiment2biom(hf3,qiimeVersion=FALSE)
biom(hf4)
write_biom(hf4, "humfum_trimmed.biom")

#load filtered data set with metagenomeseq

getwd()
```


```{r prevalence threshold, echo=FALSE}
ggplot(prevdf, aes(TotalAbundance, Prevalence, color = Class)) +
  geom_hline(yintercept = prevalenceThreshold, alpha = 0.5, linetype = 2) +
  geom_point(size = 2, alpha = 0.7) +
  scale_y_log10() + scale_x_log10() +
  xlab("Total Abundance") +
  facet_wrap(~Phylum)
```


```{r}
#bacteria
hb<-import_biom("/Users/sonyaerlandson/Documents/Chapter3/Data/otu_table_wTax16S.biom", treefilename = "/Users/sonyaerlandson/Documents/Chapter2/Data/phylogeny16S.tre" ,parseFunction = parse_taxonomy_greengenes) 


#match samples with bacteria otu table
samplenamesbac<-sample_names(hb)
shb<-as.character(sampledata$ID) %in% sample_names(hb)    
samb<-sampledata[shb, ]
row.names(samb)<-samb$ID
samb<-data.frame(samb)
sample_data(hb)<-samb
hb

unique(tax_table(hb)[ ,1]) #remove NA
kingdom<-as.vector(data.frame(tax_table(hb))$Kingdom)
unique(kingdom)
tax.rm<-kingdom == "NA"
tax.keep<-tax.rm == "FALSE"
tax.keep[is.na(tax.keep)]<-FALSE
hb0<-prune_taxa(tax.keep,hb)
taxnameshb0<-taxa_names(hb0)
taxmatch<-taxnameshb0 %in% c("OTU_1", "OTU_2", "OTU_7", "OTU_4")
unique(taxmatch)
taxmatchtrue<-taxmatch == FALSE
hb0<-prune_taxa(taxmatchtrue, hb0 )
tax_table(hb0)<-tax_table(hb0)[ ,-8]
hb0
```
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 7675 taxa and 134 samples ]
sample_data() Sample Data:       [ 134 samples by 7 sample variables ]
tax_table()   Taxonomy Table:    [ 7675 taxa by 8 taxonomic ranks ]

> hb0
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 7669 taxa and 134 samples ]
sample_data() Sample Data:       [ 134 samples by 7 sample variables ]
tax_table()   Taxonomy Table:    [ 7669 taxa by 7 taxonomic ranks ]

```{r}

# Execute prevalence filter, using `prune_taxa()` function
#hb1 = prune_taxa((prev0 >= prevalenceThreshold), hb0)
#hb1
sort(sample_sums(hb0))
hb1<-prune_samples(sample_sums(hb0)>1000, hb0)


hb2<-phyloseq_to_metagenomeSeq(hb1)
hb3<-MRexperiment2biom(hb2,qiimeVersion=TRUE)
hb3
biom(hb3)
write_biom(hb3, "humbacteria.biom")

rank_names(hb1)
names(sort(taxa_sums(hb1), decreasing = TRUE)[1:20])

tax_table(hb1)[ names(sort(taxa_sums(hb1), decreasing = TRUE)[1:20]), ]
hb.phyla<-tax_glom(hb1, taxrank = "Phylum")
hb.class<-tax_glom(hb1, taxrank = "Class")

hbotu.phyla<-data.frame(otu_table(hb.phyla))
row.names(hbotu.phyla)<-tax_table(hb.phyla)[ ,2]
write.csv(hbotu.phyla, "bacteria_phyla_otutable.csv")

hbotu.class<-data.frame(otu_table(hb.class))
row.names(hbotu.class)<-tax_table(hb.class)[ ,3]
write.csv(hbotu.class, "bacteria_class_otutable.csv")

hb3<-MRexperiment2biom(hb2.phyla,qiimeVersion=TRUE)
hb3
biom(hb3)
write_biom(hb3, "humbacteria.biom")



plot_tree(hb1)
```
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 4635 taxa and 134 samples ]
sample_data() Sample Data:       [ 134 samples by 7 sample variables ]
tax_table()   Taxonomy Table:    [ 4635 taxa by 7 taxonomic ranks ]

> hb2
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 4635 taxa and 133 samples ]
sample_data() Sample Data:       [ 133 samples by 7 sample variables ]
tax_table()   Taxonomy Table:    [ 4635 taxa by 7 taxonomic ranks ]
